doc/userGuide.md

```diff
@@ -744,15 +744,15 @@ Final configuration:
 -   **Line 10**: The `ListControl` has two _action capabilities_, called "set" and
     "change". There may be different verbs associated with these capabilities and line 10
     shows how to set the list of verbs associated with the "set" capability. This line
-    declares that if the user says "set", "add", or "select", or one of their synonyms,
+    declares that if the user says "set", "select", or one of their synonyms,
```

Comment:
Ah, I missed this. My mind saw the description of the defaults for ListControl 2 bullet points up and assumed it was the same as the code in the description for line 5. Do you think it might be clearer if we say "This line in the definition for the `CategoryControl` declares that if the user says "set", "add", or "select"..."? Or maybe add a comment in the code like `// Defaults for ListControl`? This could very well just be a me-problem.